Plugins/Weapon/Weapon.hpp

```diff
@@ -13,6 +13,20 @@
 #define NWNX_WEAPON_OPT_GRTFOCUS_AB_BONUS 0
 #define NWNX_WEAPON_OPT_GRTSPEC_DAM_BONUS 1
 
+#define NWNX_WEAPON_GETDATA_DC_WEAPON  0
+#define NWNX_WEAPON_GETDATA_DC_TARGET  1
```

Comment:
Preexisting, but I'd say favor `const int` or `constexpr int` over defines. This can be confusing, but consts in C++ are different than consts in C, they are much more like defines with type and syntax safety.